sbws/lib/destination.py

```diff
@@ -179,8 +179,7 @@ def _perform_usability_test(self):
         session = requests_utils.make_session(cont, timeout)
         usable_dests = []
         for dest in self._all_dests:
-            possible_exits = [e for e in self._rl.exits
-                              if e.can_exit_to(dest.hostname, dest.port)]
+            possible_exits = [e for e in self._rl.exits]
```

Comment:
**If** it's a good idea to stop checking exit policies against the destination we want to use, then this should just be `possible_exits = self._rl.exits`.

You'll avoid errors caused by DNS failing (yay?) but will induce errors by trying to exit Tor from an exit that doesn't support our destination (which could have been prevented).

Reply:
i'm actually implementing for checking if we're allow to exit without having to solve dns